.buildkite/bench-restore.sh

```diff
@@ -1,5 +1,5 @@
 #! /usr/bin/env nix-shell
-#! nix-shell -i bash -p coreutils gnugrep gawk time haskellPackages.hp2pretty buildkite-agent gnuplot
+#! nix-shell -i bash -p nix coreutils gnugrep gawk time haskellPackages.hp2pretty buildkite-agent gnuplot
```

Comment:
```suggestion
#! nix-shell -i bash -p nix coreutils gnugrep gawk time haskellPackages.hp2pretty buildkite-agent gnuplot
```

🤪 This will probably need to be changed for the other scrips as well?

Reply:
I removed the change, actually. Let's see what buildkite says